linode/manager
# upcoming: [M3-8028]: Linode Create Refactor - User Defined Fields

## Description 📝

Adds initial User Defined Field support to the Linode Create flow when deploying from a StackScript

## Changes  🔄
- Extracts out some until functions from the existing create flow and unit tests them (no changes in logic, only naming changes) 🧪 
- Adds initial User Defined Field support to the new Linode Create flow

## Preview 📷

![Screenshot 2024-04-22 at 3 02 13 PM](https://github.com/linode/manager/assets/115251059/01a1f955-db8d-4d3c-8b14-3d6e2ddd9e5c)


## How to test 🧪

### Prerequisites
- Check out this PR or use an internal preview link
- Enable the `Linode Create v2` feature flag

### Verification steps
- Select various community StackScripts and check the functionality of the user defined fields section
- Compare behavior to production

## As an Author I have considered 🤔
